tests/helpers/strings.spec.js

```diff
@@ -112,4 +112,134 @@ describe('strings', () => {
             })).toEqual('Hello Kabir! How are you?');
         });
     });
+
+    describe('lowercase', () => {
+        it('lowercase should return lowercase value of a string param', () => {
+            expect(strings.lowercase('Hello World!')).toEqual('hello world!');
+        });
+
+        it('lowercase should should work as expected after compilation (Basic Support)', () => {
+            var template = compile('{{lowercase val}}');
+            var obj = {
+                val: 'JUST WOW!!!'
+            };
+
+            expect(template(obj)).toEqual('just wow!!!');
+        });
+    });
+
+    describe('uppercase', () => {
+        it('uppercase should return uppercase value of a string param', () => {
+            expect(strings.uppercase('hello world!')).toEqual('HELLO WORLD!');
+        });
+
+        it('uppercase should work as expected after compilation (Basic Support)', () => {
+            var template = compile('{{uppercase val}}');
+            var obj = {
+                val: 'just wow!!!'
+            };
+
+            expect(template(obj)).toEqual('JUST WOW!!!');
+        });
+    });
+
+    describe('first', () => {
+        it('first should return first element of an array(string)', () => {
+            expect(strings.first(['David', 'Miller', 'Jones'])).toEqual('David');
+        });
+
+        it('first should work as expected after compilation (Basic Supprt)', () => {
+            var template = compile('{{first fullName}}');
+            var obj = {
+                fullName: [
+                    'David',
+                    'Miller',
+                    'Jones'
+                ]
+            };
+
+            expect(template(obj)).toEqual('David');
+        });
+    });
+
+    describe('last', ()=> {
+        it('last should return last element of an array(string)', () => {
+            expect(strings.last(['David', 'Miller', 'Jones'])).toEqual('Jones');
+        });
+
+        it('last should work as expected after compilation (Basic Supprt)', () => {
+            var template = compile('{{last fullName}}');
+            var obj = {
+                fullName: [
+                    'David',
+                    'Miller',
+                    'Jones'
+                ]
+            };
+
+            expect(template(obj)).toEqual('Jones');
+        });
+    });
+
+
+    describe('concat', () => {
+        it('concat should return concatenation of all param values(string)', () => {
+            expect(strings.concat('hello', ' ', 'world', '!!!')).toEqual('hello world!!!');
+        });
+
+        it('concat should return concatenation of all param values(string and integer)', () => {
+            expect(strings.concat('I have got', ' ', 4, ' ', 'apples.')).toEqual('I have got 4 apples.');
+        });
+
+        it('concat should work as expected for string params after compilation (Basic Support)', () => {
+            var template = compile('{{concat first \' \' middle \' \' last}}');
```

Comment:
you can use double quotes inside the string here. :)
eg: `var template = compile('{{concat first " " middle " " last}}');`